docs/manual/layout/site-structure/configure-pages.de.md

```diff
@@ -77,6 +73,124 @@ einer eindeutigen Beschreibung zu versehen.
+Aliase auszuwählen.
+
+| Alias-Einstellungen                  | Erklärung                                                            |
+|:-------------------------------------|:---------------------------------------------------------------------|
+| Unicode-Zahlen und -Kleinbuchstaben  | Aus dem Seitennamen »Über uns« wird das Alias `über-uns` generiert.  |
+| Unicode-Zahlen und -Buchstaben       | Aus dem Seitennamen »Über uns« wird das Alias `Über-uns` generiert.  |
+| ASCII-Zahlen und -Kleinbuchstaben    | Aus dem Seitennamen »Über uns« wird das Alias `ueber-uns` generiert. |
+| ASCII-Zahlen und -Buchstaben         | Aus dem Seitennamen »Über uns« wird das Alias `Ueber-uns` generiert. |
+
+Für die Erzeugung des Aliases ist in Einzelfällen auch die eingestellte Sprache relevant. So wird ein Deutsches »Über« 
+zu »ueber« jedoch ein Finnisches »eläinkö« zu »elainko« konvertiert.
+
+{{% notice note %}}
+Diese Einstellung befindet sich in Contao **4.5** bis **4.9** in der Sektion _Alias-Einstellungen_.
+{{% /notice %}}
+
+{{< version "4.10" >}}
+
+**Ordner-URLs verwenden:** Wenn eine neue oder bestehende Seite ohne eingegebenen Alias abgespeichert wird, wird der Alias
+von Contao anhand des Seitennamens automatisch generiert. Ist diese Einstellung aktiv, wird bei der automatischen Generierung
+auch die Seitenhierarchie berücksichtigt. Dabei werden die Aliase der übergeordneten Seiten durch Slashes getrennt an den
+Alias der Seite vorangestellt.
+
+Diese Funktion kann in Contao **4.10** pro Startpunkt definiert werden. In älteren Contao Versionen wird diese Funktion
+noch über [eine Systemeinstellung][SystemSettingFolderUrl] aktiviert.
+
+
+#### Legacy Routing Modus
+
+{{< version "4.10" >}}
+
+Die Einstellungen **URL-Präfix** und **URL-Suffix** so wie **Sprachweiterleitung deaktivieren** sind erst verfügbar, wenn
+das sogennante »Legacy Routing« über die Contao `contao.legacy_routing` Konfigurationseinstellung deaktiviert wurde.
+Andernfalls wird die URL-Generierung nach wie vor nur durch die Einstellungen `contao.prepend_locale` und `contao.url_suffix` 
+bestimmt.
+
+```yml
+# config/config.yml
+contao:
+    legacy_routing: false
+```
+
+**Beachte allerdings**, dass die Deaktivierung des Legacy Routing Modus auch folgende Hooks deaktiviert:
+
+* [`getRootPageFromUrl`][GetRootPageFromUrlHook]
+* [`getPageIdFromUrl`][GetPageIdFromUrlHook]
+
+{{% notice warning %}}
+Falls Extensions installiert sind, die diese Hooks noch benötigen, dann muss entweder das Legacy Routing aktiviert bleiben,
+oder die Extensions müssen entfernt oder ersetzt werdne. Andernfalls wird ein Fehler im Frontend auftreten.
+{{% /notice %}}
+
+
+### Spracheinstellungen
+
+**Sprache:** Hier kannst du die Sprache des Startpunkts festlegen. Sprachen werden über ihr primäres 
+[Subtag](http://ssgfix.sub.uni-goettingen.de/projekt/doku/sprachcode.html) nach ISO 639-1 erfasst, also z. B. über 
```

Comment:
I usually link to wikipedia, but totally up to you 🙃 
https://de.wikipedia.org/wiki/Liste_der_ISO-639-1-Codes

Reply:
That's how it is in the current documentation. We can also change this in a separate commit later.